src/legacy/ui/public/vislib/components/zero_injection/flatten_data.js

```diff
@@ -19,30 +19,26 @@
 
 import _ from 'lodash';
 
-export function VislibComponentsZeroInjectionFlattenDataProvider() {
```

Comment:
Thanks for pulling out all of these unused providers! I've had this in my mental to-do list for awhile now, so appreciate you taking the time to actually do it.